cmake/modules/Libdispatch.cmake

```diff
@@ -77,6 +77,8 @@ foreach(sdk ${DISPATCH_SDKS})
                           -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
                           -DCMAKE_SYSTEM_NAME=${SWIFT_SDK_${sdk}_NAME}
                           -DCMAKE_SYSTEM_PROCESSOR=${arch}
+                          -DCMAKE_ANDROID_NDK="${SWIFT_ANDROID_NDK_PATH}"
+                          -DCMAKE_ANDROID_ARCH_ABI=${arch}
```

Comment:
`${arch}` (eg `aarch64`) is not always the same as ABI (eg `arm64-v8a`). This is unknown for both ARMv7 and AArch64.

Reply:
Thanks, any suggestion on how to handle this?